### ISSUE TITLE
show metadata of organizations

### ISSUE THREAD

**Issue by jiskattema** [research-software-directory/RSD-as-a-service#395]
I would expect to be able to see a page with more information on an organization.
This page: https://research-software-directory.org/organisations/netherlands-escience-center
shows the projects and software, but other metadata on the organization is not shown.

For projects, there is the grant id, start end dates, running/finished etc.
For organizations i would expect some similar info: 
* identifier
* country/region
* institute/uni/commercial status
* contact person

**Comment by jmaassen** [comment]
Good suggestion, we will work on this in a later version.

**Comment by jmaassen** [comment]
Related to #482 

Will be part of milestone 8

**Comment by cmeessen** [comment]
Another topic we discussed in the daily today are the breadcrumbs. Currently, they display the slug in the URL. This does not allow special characters, e.g.:

![image](https://user-images.githubusercontent.com/14222414/194039446-c875d9f3-795b-4ae7-a97b-c70a4dc15de3.png)

We discussed how this could be improved. A possible solution is to use abbreviations instead of the slug. This would require a field in the settings that will allow maintainers to define the abbrivation. If the abbreviation is present in the ROR database, this one could be automatically set when creating the organisation as well (see `acronyms` in https://ror.readme.io/docs/rest-api). Adding abbreviations will also allow to display them in the organisation grid if a logo does not exist, which would make them more findable as well.

Additionally, we could omit the "Organisations" at the first position, because this link is already present in the header. The result could look something like this (including browsing a research unit):

![image](https://user-images.githubusercontent.com/14222414/194040271-cfdfbefd-a09e-4d9d-9089-29cb54c93629.png)

![image](https://user-images.githubusercontent.com/14222414/194040171-937c14d0-9664-4d04-9c93-3e415081a822.png)
